src/Compilers/VisualBasic/Portable/Lowering/ExpressionLambdaRewriter/ExpressionLambdaRewriter_BinaryOperator.vb

```diff
@@ -32,7 +22,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                      BinaryOperatorKind.Divide,
                      BinaryOperatorKind.Modulo,
                      BinaryOperatorKind.IntegerDivide,
-                     BinaryOperatorKind.Concatenate,
```

Comment:
nit: If it's the same situation as the `Like`, consider moving this case below and adding to the comment (line 44)